Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
```

Comment:
### [sentry-rails](https://github.com/getsentry/sentry-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.4.2 → 5.16.1

[_[change-log](https://github.com/getsentry/sentry-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/getsentry/sentry-ruby), [gem-diff](https://my.diffend.io/gems/sentry-rails/5.4.2/5.16.1)_]





<details>
<summary>Commits</summary>

A change of **219** commits. See the full changes on [the compare page](https://github.com/getsentry/sentry-ruby/compare/5.4.2...5.16.1).

These are the first 10 commits:
- (67bb989) [Merge branch 'release/5.4.2'](https://github.com/getsentry/sentry-ruby/commit/67bb989b3c0be4f7d4098ef10f8d54ab61c42a97)
- (29f5875) [Remove webmock (#1871)](https://github.com/getsentry/sentry-ruby/commit/29f5875458d0ee2aaadb14382371ca16fdbfd4a5)
- (353a813) [Update error subscriber doc (#1880)](https://github.com/getsentry/sentry-ruby/commit/353a813378e708657ca83336947720406b4d9bc4)
- (978cc48) [Support rack 3 (#1884)](https://github.com/getsentry/sentry-ruby/commit/978cc48f3912e16698c6b517e4dc37e1a20b411c)
- (53795b7) [Raven CI still needs bundler: 1 (#1888)](https://github.com/getsentry/sentry-ruby/commit/53795b7f651e0958876630be5965e79726caf4c4)
- (67bbcd7) [Pin rack to < 3 on sentry-raven (#1889)](https://github.com/getsentry/sentry-ruby/commit/67bbcd77c0221b0abafebc8889b4f8233a7c1758)
- (d872488) [Memoize session.aggregation_key (#1892)](https://github.com/getsentry/sentry-ruby/commit/d8724882b67c3107d062456e4fc16237aa5e46f9)
- (f931fe1) [Add Ruby head to CI matrix (#1893)](https://github.com/getsentry/sentry-ruby/commit/f931fe1e7a0ff8d7f58aa0334950c8bb1719617f)
- (ae13d7d) [Execute with_scope's block even when SDK is not initialized …](https://github.com/getsentry/sentry-ruby/commit/ae13d7dbdf72c964e5b3aca78cf4682a28c21192)
- (d83b29c) [Configure Rails 7.1's test app separately (#1901)](https://github.com/getsentry/sentry-ruby/commit/d83b29c75d8fda3cb0a8e4a8d4d3bfc4bbaa0ae4)


</details>